content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,28 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
```

Comment:
I'm curious, is there any way we can present the concept of idmap mounts for newcomers that aren't familiar with this from Linux?
https://lwn.net/Articles/896255/

Reply:
How I'd cover that:
- have a blog article that explains Kubernetes and idmapped mounts
- publish that article a week or so before the release
- mark the article as `evergreen` (ie: we'll maintain it; usually we don't maintain blog articles once they reach a year old)
- link from these user namespaces docs to that evergreen blog article

An example of an evergreen article: https://kubernetes.io/blog/2020/09/03/warnings/

Reply:
IMHO that seems like too much overhead. It's not clear the value it would have to have such an ongoing investment to keep it up to date, as it is suggested. There are other places that do document idmap mounts, not sure I see the point of a k8s specific documentation.

Furthermore, Kubernetes users just need to know if a filesystem supports it or not.